docs/sections/cn_workflows.md

```diff
@@ -359,6 +359,55 @@ popper run -f sample.yml -c config.yml
 
 This runs the step `one` locally in the host and step `two` through slurm on 2 nodes.
 
+## Kubernetes
+
+Popper can be run [Kubernetes cluster](https://kubernetes.io/docs/setup/) as the resource manager to orchestrate the execution of a step to one a single node.
+You can get started with running Popper workflows through Kubernetes by following the example below.
+
+Let's consider a workflow `sample.yml` like the one shown below.
+
+```yaml
+version: '1'
+steps:
+- id: one
+  uses: docker://alpine:3.9
+  args: echo hello-world
+
+- id: two
+  uses: popperized/bin/sh@master
+  args: ls -l
+```
+
+To run all the steps of the workflow through kubernetes,
+use the `--resource-manager` or `-r` option of the `popper run` subcommand to specify the resource manager.
+
+```bash
+popper run -f sample.yml -r kubernetes
+```
+
+For a workflow we build an image and create a volume in the namespace defined in the [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). 
```

Comment:
another important point is to mention that the volume that gets created is persisted throughout the execution of the workflow.

Reply:
Oh I didn't know this. Thanks for adding this